templates/default/net/netinfo.html

```diff
@@ -73,12 +73,27 @@ <H1>{$layout.pagetitle}</H1>
 		<TD colspan="3">
 			{if $unlockedit}
 			<INPUT TYPE="text" NAME="networkdata[interface]" VALUE="{$network.interface}" {tip text="Enter interface where network is connected to (optional)" trigger="interface"}>
-			<span class="bold">{trans("vlan:")}</span>
-			<INPUT TYPE="text" NAME="networkdata[vlanid]" VALUE="{if $network.vlanid}{$network.vlanid}{/if}" SIZE="5" MAXLENGTH="4" {tip text="Enter vlan ID (optional)" trigger="vlanid"}>
-			{else}
-			{if !empty($network.interface)}{$network.interface}{/if}{if !empty($network.vlanid)}, {trans("vlan:")} {$network.vlanid}{/if}
-			{/if}
-		</TD>
+            {else}
+                {$network.interface}
+            {/if}
+        </TD>
+    </TR>
+    <TR>
+        <TD class="bold">
+            {icon name="vlan" label="VLAN:"}
+        </TD>
+        <TD colspan="3">
+            {if $unlockedit}
+            <select name="networkdata[vlanid]" class="lms-ui-advanced-select">
+                {foreach $vlanlist as $v}
+                <option value="{$v.id}" {if $network.vlanid == $v.vlanid} selected{/if}>(#{$v.vlanid}){if $v.description}, {$v.description|truncate:'20':''}{/if}{if $v.customername}, {$v.customername}{/if}</option>
+                {/foreach}
+                <option value="0" {if $network.vlanid == 0} selected{/if}>{trans("None")}</option>
+            </select>
+            {else}
+                {$network.vlanid}
+            {/if}
+        </TD>
```

Comment:
formatowanie!

Reply:
Stosuję space zamiast tabów dlatego to się rozjeżdza. Czy powinienem na siłę trzymać się starej koncepcji również w nowych commitach?

Reply:
Zauważ, że w PHPStorm kod wygląda OK.

Reply:
@chilek jak ma docelowo to wyglądać? Mają być spacje czy taby w nowych commitach?

Reply:
?

Reply:
Poprawione.
Jak jest źle to popraw bo nadal nie sprecyzowałeś się czego oczekujesz.